models/models.js

```diff
@@ -65,3 +64,40 @@ exports.fetchArticles = () => {
       return result.rows;
     });
 };
+
+exports.addComment = (articleID, newComment) => {
+  const { username, body } = newComment;
+  if (
+    username &&
+    typeof username === "string" &&
+    body &&
+    typeof body === "string" &&
+    Object.keys(newComment) !== 2
```

Comment:
```js
    Object.keys(newComment) !== 2
```
What is this condition checking for? I assume you were after the number of keys, in which case you're missing the `.length` property. However a correctly formatted comment should have exactly 2 keys.